tests/logging_callback_tests/test_logging_redaction_e2e_test.py

```diff
@@ -118,3 +118,36 @@ async def test_global_redaction_off_with_dynamic_params(turn_off_message_logging
             == "hello"
         )
         assert standard_logging_payload["messages"][0]["content"] == "hi"
+
+
+@pytest.mark.asyncio
+async def test_redaction_responses_api():
+    """Test redaction with ResponsesAPIResponse format"""
+    litellm.turn_off_message_logging = True
+    test_custom_logger = TestCustomLogger()
+    litellm.callbacks = [test_custom_logger]
+    
+    # Mock a ResponsesAPIResponse-style response
+    mock_response = {
+        "output": "This is a test response",
+        "model": "gpt-3.5-turbo",
+        "usage": {"total_tokens": 10}
+    }
+    
+    response = await litellm.acompletion(
```

Comment:
why is this calling completion if it's testing responses api? shouldn't this call litellm.aresponses ?

Reply:
Updated